src/sentry/static/sentry/app/components/dropdownAutoComplete/menu.tsx

```diff
@@ -25,6 +25,7 @@ type EventHandles = {
 
 type MenuProps = Omit<EventHandles, 'onKeyDown'> & {
   className?: string;
+  style?: React.CSSProperties;
```

Comment:
you could use the prop style...this will be forward to the menu 😉   Please see 

https://github.com/getsentry/sentry/blob/861531308c5ef4e26d8dfc0e8690b951ea17d17f/src/sentry/static/sentry/app/components/dropdownAutoComplete/menu.tsx#L185

Reply:
I think this didn't work all this time as styles in `menuProps` are being overridden later here:
https://github.com/getsentry/sentry/blob/efc222db9a30662a0527218fc473e005150370ee/src/sentry/static/sentry/app/components/dropdownAutoComplete/menu.tsx#L305-L315
I removed the `menuProps={{style: {position: 'relative'}}}` line altogether.